### PR TITLE
Fix rhel7 openshift tests by proper calling function ct_os_upload_image

### PR DESCRIPTION
**Pull request by phracek**
This pull request fixes proper calling function `ct_os_upload_image`

Container-common-scripts are also updated.

### CODE REVIEW

**Comment by phracek** [comment]
[test-openshift]

**Comment by phracek** [comment]
[test-openshift]

**Comment by phracek** [comment]
Redis 6 imagestream was added.

[test-openshift]

File: imagestreams/redis-centos.json [hunk 1]
```diff
@@ -62,6 +62,24 @@
         },
         "name": "5-el7"
       },
+      {
+        "annotations": {
+          "description": "Provides a Redis 5 database on CentOS 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/redis-container/tree/master/6/README.md.",
```
Comment (zmiklank) [inline imagestreams/redis-centos.json:67]: Redis 6, not Redis 5.

**Review by zmiklank — APPROVED** [review]
One nitpick, else LGTM, thank you.

File: imagestreams/redis-centos.json (hunk 1 above)
Comment (phracek) [reply imagestreams/redis-centos.json]: Upps, Copy paste error.

**Comment by phracek** [comment]
[test-openshift]

**Pull request merged**